src/gax.ts

```diff
@@ -209,31 +235,50 @@ export class CallSettings {
     let longrunning = this.longrunning;
     let apiName = this.apiName;
     let retryRequestOptions = this.retryRequestOptions;
+
     // If a method-specific timeout is set in the service config, and the retry codes for that
     // method are non-null, then that timeout value will be used to
     // override backoff settings.
-    if (
-      retry !== undefined &&
-      retry !== null &&
-      retry.retryCodes !== null &&
-      retry.retryCodes.length > 0
-    ) {
-      retry.backoffSettings.initialRpcTimeoutMillis = timeout;
-      retry.backoffSettings.maxRpcTimeoutMillis = timeout;
-      retry.backoffSettings.totalTimeoutMillis = timeout;
+    if (retry !== undefined && retry !== null) {
+      // if verify that the retry codes/retry function are not null or undefined
```

Comment:
nit: this comment is obvious from the code, go/comments-codelab#dont-comment-the-obvious

Reply:
LOL I am normally much better about this - this is likely leftover from an outline I made in human readable terms of what I wanted to happen